tools/compile_tf_graph.py

```diff
@@ -741,9 +911,7 @@ def main(argv):
       from TFUtil import CollectionKeys
       with open(args.output_file_state_vars_list, "w") as f:
         for param in tf.get_collection(CollectionKeys.STATE_VARS):
-          assert param.name[-2:] == ":0"
-          f.write("%s\n" % param.name[:-2])
-
-
+          f.write("%s\n" % param.name)
+ 
```

Comment:
I think there is one too less empty line here. Why did you remove that? It should be two empty lines. (PyCharm should tell you that, or not?)